### ISSUE TITLE
[BUG]  Set-PnPTenantSite not working on site collection  if connected to SharePoint Admin URL

### ISSUE THREAD

**Issue by siddharth-vaghasia** [pnp/powershell#932]
### Reporting an Issue or Missing Feature
Set-PnPTenantSite -Url https://tenant.sharepoint.com/sites/TestSiteWithoutAccess -Owners 'siddharth@tenant.onmicrosoft.com'

returning 403 forbidden url though we have connected to SharePoint Admin URL using Connect-PnPOnline

Set-PnPTenantSite : The remote server returned an error: (403) Forbidden.

Additional Note - The same thing is working fine in legacy SharePointPnPPowerShellOnline.

### Expected behavior 
Command should get executed correctly and set properties on targeted site collection

### Actual behavior 
Set-PnPTenantSite : The remote server returned an error: (403) Forbidden.

![image](https://user-images.githubusercontent.com/9557557/126525217-9ca0be87-58e7-4f0e-9c3d-7a602f3ffd88.png)

### Steps to reproduce behavior

```
Function SetUserPermission(){
    try{
        Write-Host "Setting User Permission..." -ForegroundColor Yellow 
        Connect-PnPOnline -Url https://tenant-admin.sharepoint.com/ -Credentials $Credentials
         Set-PnPTenantSite -Url https://tenant.sharepoint.com/sites/TestSiteWithoutAccess -Owners 'siddharth@tenant.onmicrosoft.com'
        Write-Host "Setting User Permission Successfull..." -ForegroundColor Yellow 
    }
    catch{
        Write-Host "Error in setting user permission'" $_.Exception.Message -ForegroundColor Red  
    }
}
```

### What is the version of the Cmdlet module you are running?
1.6.0

### Which operating system/environment are you running PnP PowerShell on?
- [ ] Windows

The same command is failing in Azure Runbook also.

**Comment by siddharth-vaghasia** [comment]
Just saw another issue which might be similar and says being fixed in yesterday 1.6.44 nightly build release, so I did update to this release and can confirm this issue still exist new release also.

**Comment by veronicageek** [comment]
Yesterday was about `Get-PnPTenantSite`. Would yours be similar to issue #889?

**Comment by siddharth-vaghasia** [comment]
@veronicageek  -yes it looks similar, the only difference I see is in the error message but seems logical...we can close this Issue and we can track it under the older one.....

**Comment by SteveWedge** [comment]
Is this issue supposed to have been resolved?  I am running the nightly release 1.7.52 and am encountering the same error:- 
"Set-PnPTenantSite : The remote server returned an error: (403) Forbidden"

**Comment by vaderj** [comment]
This issue still exists in 1.12.0
